subsys/net/lib/aws_fota/include/aws_fota_json.h

```diff
@@ -46,6 +46,8 @@ extern "C" {
  * @param[in] payload_len  The length of the provided string.
  * @param[out] job_id_buf  Output buffer for the "jobId" field from the
  *			   JobExecution data type.
+ * @param[out] schema_buf  Output buffer for the "protocol" field from the Job
```

Comment:
Why `schema`? Given the description it's should be `protocol` or something.

Reply:
It was an attempt to better reflect that this is something we get out from the URL https://en.wikipedia.org/wiki/URL#Syntax so it should be named `scheme` not `schema`. But I'm fine with having it named `protocol` also.

Reply:
Personally I find `protocol` way more descriptive. I see that wiki page uses `scheme` naming, but I've found `protocol` used as well on the very same page, as well on other pages. Up to you which to use.